src/main/java/com/twilio/oai/AbstractTwilioGoGenerator.java

```diff
@@ -78,6 +78,15 @@ public void processOpenAPI(final OpenAPI openAPI) {
                 param.addExtension(ACCOUNT_SID_VEND_EXT, true);
             });
 
+        String domain = StringHelper.toSnakeCase(twilioCodegen.getDomainFromOpenAPI(openAPI));
+        int lastIndex = domain.lastIndexOf("_api");
```

Comment:
add a comment here that we are skipping _api for backward compatibility of folder names.

Reply:
Got it

Reply:
Done